### DISCUSSION TITLE
Insert metadata into body of template?

### DISCUSSION THREAD

**Discussion by GhostNite** [SilentVoid13/Templater#936] [Help] [answered]
Forgive me as I'm not familiar with JS and VERY new to Templater.

I have a template right now for uni coursework where when I create a note under certain classes folders it prompts me for responses that get cached in the metadata of the file, which then gets added to the calendar from the Projects plugin.

Is there a way I can pull those responses from the metadata system suggester and into the body of the note without having to manually type it out? Would it be best to utilize a JS script for this and have it run when the template is used?

I tried looking through the discussions but did not see anything that seemed to match up with what I am looking for.

**Comment by Zachatoo** [comment, marked as answer]
You can reuse the value from the suggester. I have an example here of how to do it with `prompt`, works well with `suggester` too.
https://zachyoung.dev/snippets/templater#reuse-value-from-prompt-or-suggester
